packages/ListBox/src/components/Content/Content.js

```diff
@@ -54,27 +41,31 @@ const handleContentFocusChange = (hasFocus, dispatch) =>
   dispatch({ type: useListBox.types.setListBoxHasFocus, payload: { hasFocus } });
 
 export default function Content(props) {
-  const { children, hasOptions } = props;
+  const { children, hasOptions, onCancelFooter, ...moreProps } = props;
   const onChangeContext = React.useContext(OnChangeContext);
   const [state, dispatch] = useListBox();
   const { refListBoxContainer } = state;
+  const providedProps = React.useContext(PropsContext);
+  const { idListBox, isDisabled, isInline } = providedProps;
 
-  /* NOTE no idea what ROLE should be this div when the ListBox is INLINE */
-  if (state.isInline) {
+  if (isInline) {
     return (
       <sc.Content
-        {...getDOMAttributesForListBoxContainer({ isInline: true })}
-        onFocus={() => {
-          handleContentFocusChange(true, dispatch);
-        }}
+        {...moreProps}
+        {...getDOMAttributesForListBox({ idListBox, ...state })}
+        data-pka-anchor="list-box-content-inline" // TODO: rename "list-box.content-inline"
```

Comment:
I wanted to rename a lot of the anchors, but decided not to introduce potentially breaking changes in this PR.

Reply:
would be hard to rename the anchors :P, if someone is relying on them to do some css or calculation could be a hard bug to catch.